### PR TITLE
Switch to quadrature for integration

### PR DESCRIPTION
**Pull request by tvatter**

### CODE REVIEW

File: include/vinecopulib/misc/implementation/tools_integration.ipp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    case 30:
+      output = tools_legendre::nodes_weights_30;
+      break;
+    case 40:
+      output = tools_legendre::nodes_weights_40;
+      break;
+    case 50:
+      output = tools_legendre::nodes_weights_50;
+      break;
+    default:
```
Comment (tnagler) [inline include/vinecopulib/misc/implementation/tools_integration.ipp]: I assume that we will always just use the default `n` (currently 20). I'd vote to keep the code base lean by removing the option altogether, possibly changing the default to something a bit larger. Did you do some accuracy checks?

Comment (tnagler) [reply include/vinecopulib/misc/implementation/tools_integration.ipp]: Started a preliminary implementation of conditional distributions. Seems like we need **at least** `n = 50`. Relative error was still around 5-10% :(

Comment (tvatter) [reply include/vinecopulib/misc/implementation/tools_integration.ipp]: Yeah, that was at that point that I stopped: I could not get the errors at a acceptably low level :(

**Comment by tnagler** [comment]
OK, how about we put this on hold and reconsider for the next release?

**Pull request closed without merging**